electroncash_plugins/email_requests/qt.py

```diff
@@ -179,17 +392,21 @@ def new_invoice(self):
         #window.update_invoices_list()
 
     @hook
-    def receive_list_menu(self, menu, addr):
+    def receive_send_via_email_list_menu(self, menu, addr, uri, qrcode, status):
         window = menu.parentWidget().parent  # Grr. Electrum programmers overwrote parent() method.
```

Comment:
To match the API I proposed above:

```suggestion
    def receive_list_menu(self, window, menu, addr):
        uri = window.get_request_URI(addr)
        qrcode = window.receive_qr
```